auth/login/login.css

```diff
@@ -0,0 +1,43 @@
+.main-body {
+  box-sizing: border-box;
```

Comment:
This can be added to all elements using 

```
* {
box-sizing: border-box;
```

If you are using component library, this should be already applied if you are having single entry point in css for all the files

Reply:
Thanks for suggestion, I'll do the changes